test/bddtests/did_sidetree_steps.go

```diff
@@ -840,9 +904,23 @@ func (d *DIDSideSteps) validateResolutionResult(url string) error {
 	}
 
 	var expected document.ResolutionResult
-	err = json.Unmarshal(body, &expected)
-	if err != nil {
-		return err
+	if url == "https://raw.githubusercontent.com/decentralized-identity/sidetree/master/tests/fixtures/longFormDid/resultingDocument.json" {
+
+		// temporary struct until they fix issue #847
```

Comment:
Do we have an issue for this as well?

Reply:
Yes, I opened https://github.com/decentralized-identity/sidetree/issues/871. When they fix it there will be no need for if statement, it will be handled as any other resolution result file.